xrpl/asyncio/account/transaction_history.py

```diff
@@ -29,12 +30,12 @@ async def get_latest_transaction(account: str, client: Client) -> Response:
     )
     if not response.is_successful():
         result = cast(Dict[str, Any], response.result)
-        raise XRPLRequestFailureException(result)
+        raise exception.XRPLRequestFailureException(result)
```

Comment:
Why do we need to change how we import exceptions?

Reply:
just a workaround to avoid circular imports by not importing specifics